### DISCUSSION TITLE
Download only representative assembly version using ncbi-genome-download

### DISCUSSION THREAD

**Discussion by KinSimon96** [kblin/ncbi-genome-download#218] [General]
Hello,

using `ncbi-genome-download --formats fasta,assembly-report bacteria --genera "Bacillus anthracis"`, it downloads all assembly versions available, which is not required for my purpose. 

[Here](https://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Bacillus_anthracis/), you can see the "representative" section, including only 1 version. Is there a way to download this one using ncbi-genome-download ?

Thanks,
Simon

**Comment by kblin** [comment]
I realise this isn't super clear from the `--help` output, but you should be able to use `--refseq-categories representative` for this.
